crates/matrix-sdk/src/event_cache/room/events.rs

```diff
@@ -185,6 +188,12 @@ impl RoomEvents {
         self.chunks_updates_as_vectordiffs.take()
     }
 
+    /// Get a mutable reference to the [`LinkedChunk`] updates, aka
+    /// [`ObservableUpdates`].
+    pub fn updates(&mut self) -> &mut ObservableUpdates<Event, Gap> {
```

Comment:
Please reduce the visiblity of this:

```diff
- pub
+ pub(super)
```

?

Reply:
Indeed, thanks!